### DISCUSSION TITLE
Settig for "Automatic edge color" in Format / Preferences Menu

### DISCUSSION THREAD

**Discussion by burnuser** [freeplane/freeplane#433] [Ideas]
Setting for "Automatic edge color - deactivate" can only be found in Tool panel. (Where I didn't search for it a long time.)

Maybe it can be included in the Format menu too?

Or even in Preferences to deactivate it generally?

**Comment by macmarrum** [comment]
Yep, it would be great to have it in `Format->Automatic edge color`

* for branches
* for levels
* for columns
* on branch creation
* disabled

And while considering this, a menu entry for `Edit edge colors` could be added too.

This would make the options available in Command Search 😄
